config/slam_toolbox_params_webots.yaml

```diff
@@ -0,0 +1,67 @@
+slam_toolbox:
+  ros__parameters:
+    # Plugin params
+    solver_plugin: solver_plugins::CeresSolver
+    ceres_linear_solver: SPARSE_NORMAL_CHOLESKY
+    ceres_preconditioner: SCHUR_JACOBI
+    ceres_trust_strategy: LEVENBERG_MARQUARDT
+    ceres_dogleg_type: TRADITIONAL_DOGLEG
+    ceres_loss_function: None #HuberLoss
+
+    # ROS Parameters
+    odom_frame: odom
+    map_frame: map
+    base_frame: base_link
+    scan_topic: /scan
+    mode: mapping #localization
+
+    debug_logging: false
+    throttle_scans: 1
+    transform_publish_period: 0.04
+    map_update_interval: 1.0
+    resolution: 0.05
+    max_laser_range: 12.0 #for rastering images
+    minimum_time_interval: 0.05
+    transform_timeout: 0.1
+    tf_buffer_duration: 20.0
+    stack_size_to_use: 40000000 #// program needs a larger stack size to serialize large maps
+    enable_interactive_mode: false
+
+    # General Parameters
+    use_scan_matching: false
```

Comment:
still not fixed? :D It would be nice to add a comment that this parameter is changed, and a general one at the beginning of the file, explaining why it is necessary to have a separate config. Also a link to the issue will be nice, so that we can keep track of it, and remove this config once it is solved.

Reply:
Added the comment at the beginning of the file.